packages/@vue/cli-plugin-pwa/README.md

```diff
@@ -53,6 +53,34 @@ file, or the `"vue"` field in `package.json`.
 
   - Default: `'default'`
 
+- **pwa.assetsVersion**
+  
+  - Default: `''`
+
+    This option is used if you need to add a version to your incons and manifest, against browser’s cache. This will append `?v=<pwa.assetsVersion>` to the URLs of the icons and manifest.
+
+- **pwa.manifestPath**
```

Comment:
We have to think about how to manage this in the UI, the PWA configuration screens attempts to load and write to this file. If the user changes the location, it will break it.